contracts/lib/OrderCombiner.sol

```diff
@@ -359,6 +390,13 @@ contract OrderCombiner is OrderFulfiller, FulfillmentApplier {
             }
         }
 
+        // If the second bit is set in the error buffer, we are not in a match function.
+        // If the first bit is set, a native offer item was encountered.
+        // If the value is greater than two, both the first and second bits were set.
```

Comment:
This comment is a bit misleading since it's true here because `> 3` is not possible but the statement isn't true in general.

Reply:
For reference, this has been fixed in `main`. comment now states:

```
        // If the first bit is set, a native offer item was encountered. If the
        // second bit is set in the error buffer, the current function is not
        // matchOrders or matchAdvancedOrders. If the value is three, both the
        // first and second bits were set; in that case, revert with an error.
```